lib/api/v3/relations/relation_representer.rb

```diff
@@ -52,6 +52,18 @@ class RelationRepresenter < ::API::Decorators::Single
           }
         end
 
+        link :toType do
+          {
+            href: api_v3_paths.type(represented.to.type_id)
+          }
+        end
+
+        link :fromType do
+          {
+            href: api_v3_paths.type(represented.from.type_id)
+          }
+        end
```

Comment:
I could be wrong but it looks like the code is no longer/not using those links. If so, I would be glad as they shouldn't be here IMO.